src/components/Details/VmDetails/tests/VmDetails.test.js

```diff
@@ -62,6 +62,11 @@ describe('<VmDetails />', () => {
     expect(component).toMatchSnapshot();
   });
 
+  it('renders correctly with services', () => {
+    const component = render(testVmDetails(vmFixtures.downVm, { services }));
```

Comment:
it only show the loading. Can you please put some rendered services?

Reply:
fixed, fixture is now rendering services